CHANGELOG.md

```diff
@@ -0,0 +1,230 @@
+# Changelog
+
+All notable changes to this project will be documented in this file - formatted and maintained 
+according to the rules documented on <http://keepachangelog.com>.
+
+This file will not cover changes about documentation, code clean-up, samples, or the CI pipeline.
+With each version (respectively milestone), the core features are highlighted. Relevant changes to
+the core and existing implementations can be found at the beginning of each section. Important 
+details are referred to by linking pull requests or issues.
+
+## [Unreleased] - XXXX-XX-XX (Milestone 3)
+
+### Added
+* `ContractDefinitionStore` supports paging (#717)
+* Add okhttp client timeouts (#735)
+
+### Changed
+
+### Removed
+* Remove ION extension (#664)
+
+### Fixed
+
+### Management API
+...
+
+### Data Plane Framework
+...
+
+### IDS
+
+#### Added
+
+#### Changed
+* Change scope used for obtaining a token for ids multipart (#731)
+* Refactor ids token validation as extension (#625)
```

Comment:
How about something like this:

```markdown
## [0.2.0] - 2021-02-21 (Milestone 2)

### Overview
- DataPlaneFramework (DPF) - first working version
- Data Management API - controller stubs and OpenApi Spec
- Code Quality: Coverage Analysis, System Test Framework
- Traceability and Metrics

### Detailed Changes

#### Added
* `ContractDefinitionStore` supports paging (#717)
* Add okhttp client timeouts (#735)

#### Changed
* Change scope used for obtaining a token for ids multipart (#731)
* Refactor ids token validation as extension (#625)

#### Removed
* Remove ION extension (#664)

#### Fixed

```

Reply:
@paullatzelsperger Could you pls take another look? I also added some details to the CONTRIBUTING.md.

Reply:
nice, looks good. can be merged IMO.